matrix-sdk-android/src/main/java/org/matrix/android/sdk/internal/session/room/relation/threads/FetchThreadSummariesTask.kt

```diff
@@ -44,7 +44,7 @@ internal interface FetchThreadSummariesTask : Task<FetchThreadSummariesTask.Para
     data class Params(
             val roomId: String,
             val from: String = "",
-            val limit: Int = 100,
+            val limit: Int = 500,
```

Comment:
Why 500 as default?

Reply:
Until backend will support pagination, we concluded to increase the limit here